test/Cronofy.Test/CronofyOAuthClientTests/GetAuthorizationUrl.cs

```diff
@@ -27,8 +27,8 @@ public void HasDefaultScope()
                 "https://app.cronofy.com/oauth/authorize" +
                     "?client_id={0}" +
                     "&response_type=code" +
-                    "&scope=read_account%20read_events%20create_event%20delete_event" +
-                    "&redirect_uri={1}",
+					"&redirect_uri={1}" +
```

Comment:
This has been entabbed like the other stuff. We don't have StyleCop on the tests as the readability of the code is more important than strict formatting rules.

Good luck resolving this without making a mess. I'd suggest not doing it in Xamarin.